ekncontent/docs/reference/ekncontent/eos-knowledge-content-sections.txt

```diff
@@ -130,3 +130,22 @@ EKNC_SUBTREE_DISPATCHER_GET_CLASS
 <SUBSECTION Private>
 eknc_subtree_dispatcher_get_type
 </SECTION>
+
+
+<SECTION>
+<FILE>xapian-bridge</FILE>
+eknc_xapian_bridge_get_fixed_query
+eknc_xapian_bridge_get_fixed_query_finish
+eknc_xapian_bridge_query
+eknc_xapian_bridge_query_finish
+eknc_xapian_bridge_get_soup_session
```

Comment:
I was fine with exposing, might be useful exposed down the line. But if we want to exposed only the smallest surface possible I can keep it out of the docs.